lib/perl/Genome/Model/Build/SomaticVariation.pm

```diff
@@ -484,4 +484,29 @@ sub build_for_tumor_bam {
     return $self->tumor_build;
 }
 
+sub _disk_usage_result_subclass_names {
+    my $self = shift;
+
+    my @disk_usage_result_classes = (
+        'Genome::InstrumentData::AlignmentResult::BwamemStream',
```

Comment:
Ignore this. alignments should be in RefAlign. We should however remove it from the list then.